spec/features/guidance_after_sign_in_spec.rb

```diff
@@ -3,44 +3,52 @@
 
 describe 'guidance after sign in' do
   let(:user) { create(:user, :confirmed, :with_organisation) }
-  let!(:location) { create(:location, organisation: user.organisation) }
+  before { sign_in_user user }
 
-  before do
-    sign_in_user user
-    visit '/'
-  end
-
-  it 'shows me the landing guidance' do
-    expect(page).to have_content 'Get GovWifi'
-    expect(page).to have_content 'Getting help'
-  end
+  context 'without locations' do
+    before { visit root_path }
 
-  it 'displays radius connection details' do
-    expect(page).to have_content(location.radius_secret_key)
+    it 'displays message to inform user to add IPs and locations' do
+      expect(page).to have_content 'Your Radius secret keys will be generated when you add your first IP address'
```

Comment:
I worry that testing against this specific message is brittle - I've tended to try to and check against a short part of these descriptions, e.g. `RADIUS secret keys will be generated`, or maybe not even testing the message at all.

Happy for you to make a judgement call on this.